.github/ISSUE_TEMPLATE/give_a_talk.yml

```
@@ -0,0 +1,36 @@
+name: Give a TTW talk 
+description: Resources and checklist for giving a talk about the Turing Way
+title: "[TALK] <title>"
+labels: [talks-and-workshops, newsletter]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This template will guide you through the steps necessary to give a successful talk about The Turing Way!
+  - type: input
+    id: date
+    attributes:
+      label: Date of talk
+      description: When will you be presenting about the Turing Way?
+      placeholder: please use the format YYYY-MM-DD
+    validations:
+      required: true
+  - type: textarea
+    id: talk_details
+    attributes:
+      label: Details of the talk
+      description: Where are you presenting? Please provide a link to the event if available
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Checklist
+      description: Access TTW slides, check acknowledgements, and share your presentation
+      options:
+        - label: Schedule your talk and let the community know in TTW Slack. If you’d like feedback and/or to schedule a practice talk, ask in the TTW Slack!
+        - label: Download template(s) from the [promotion pack](https://drive.google.com/drive/folders/1mzGmbJkPnP5q1goQesxDc_E5zAPL0eTF?usp=sharing) to ensure stylistic concistency
+        - label: If you re-used slides from a specific talk, please acknowledge the original author of the slides 
+        - label: Double check acknowledgements slide for TTW team, license info, Scriberia link, any additional acknowledgements
+        - label: Double check contact info for TTW links (book, Twitter, GitHub, Slack, newsletter), your own contact into
+        - label: Upload your slides to [Zenodo](zenodo.org/), tag with the-turing-way under "Communities". Zenodo allows [versioning](https://help.zenodo.org/#versioning), so we encourage you to upload your slides before your talk and update the file with any changes after.
```

```suggestion
        - label: Upload your slides to [Zenodo](zenodo.org/), preferably in the original format along with a PDF or any other format you are using.During the upload, tag with the-turing-way under "Communities". Zenodo allows [versioning](https://help.zenodo.org/#versioning), so we encourage you to upload your slides before your talk and update the file with any changes after.
```

Just making this point a little more specifc. WDYT?

---

we were working on this at the same time! it's been updated